src/bp-templates/bp-nouveau/includes/media/functions.php

```diff
@@ -129,7 +129,7 @@ function bp_nouveau_media_localize_scripts( $params = array() ) {
 		'group_album'                        => bp_is_group_albums_support_enabled(),
 		'messages_media'                     => bp_is_messages_media_support_enabled() && bb_user_can_create_media(),
 		'messages_media_active'              => bp_is_messages_media_support_enabled(),
-		'dropzone_media_message'             => __( 'Drop images here to upload', 'buddyboss' ),
+		'dropzone_media_message'             => __( '<strong>Add Photos</strong> Or drag and drop', 'buddyboss' ),
```

Comment:
Need to use sprintf

Reply:
@chetansatasiya This is fixed. Please check